src/aliases/store/store.go

```diff
@@ -0,0 +1,50 @@
+package aliasstore
+
+import "context"
+
+//go:generate mockgen -source=store.go -destination=mock/store.go -package=mock
+
+// Store handles the alias storing.
+type Store interface {
```

Comment:
`Store` is your `Alias` agent that should be appended to the `database.Database` interface in the same way as `ETH1Accounts`, `Keys` and `Secrets`. We can have a call to peer on this to have a consistent way of defining agents.

Reply:
We decided to keep the database a direct dependency of each service, so no global Database interface should group every service datastore together.

Reply:
This is your `Alias` interface. Because of the refactor, you don't need to add `Alias` to all your function names, they can just be `Create, Get, etc.`